docs/TabbedForm.md

```diff
@@ -824,55 +824,54 @@ Check [the `<AutoSave>` component](./AutoSave.md) documentation for more details
 
 ## Role-Based Access Control (RBAC)
 
-Fine-grained permissions control can be added by using the [`<TabbedForm>`](./AuthRBAC.md#tabbedform) and [`<FormTab>`](./AuthRBAC.md#tabbedform) components provided by the `@react-admin/ra-rbac` package. 
+Fine-grained permissions control can be added by using the [`<TabbedForm>`](./AuthRBAC.md#tabbedform) component provided by the `@react-admin/ra-rbac` package.
+
+Use in conjunction with [`<TabbedForm.Tab>`](#tabbedformtab) and add a `name` prop to the `Tab` to define the resource on which the user needs to have the 'write' permissions for.
```

Comment:
```suggestion
Use in conjunction with ra-rbac's [`<TabbedForm.Tab>`](#tabbedformtab-1) and add a `name` prop to the `Tab` to define the resource on which the user needs to have the 'write' permissions for.
```

There is already a section called `<TabbedForm.Tab>` in this doc. We need to be more specific about which one we speak of.